app/messaging/schemas/comms-message.js

```diff
@@ -0,0 +1,15 @@
+import Joi from 'joi'
+
+const schema = Joi.object({
```

Comment:
Consider changing file name to match class name to avoid code smell

Reply:
Function missing unit test

Reply:
@AbidemiAdio which function exactly?